python/guide/module/gluon/nn.md

```diff
@@ -151,25 +161,29 @@ Indeed, it is no different than It can observed here that the use of the
 `MySequential` class is no different from the use of the Sequential class.
 
 
-##
-Blocks with Code
+## Blocks with Code
 
-Although the Sequential class can make model construction
-easier, and you do not need to define the `forward` method, directly inheriting
-the Block class can greatly expand the flexibility of model construction. In
-particular, we will use Python's control flow within the forward method. While
-we're at it, we need to introduce another concept, that of the *constant*
-parameter. These are parameters that are not used when invoking backprop. This
-sounds very abstract but here's what's really going on. Assume that we have some
-function
+### Blocks
+The `Sequential` class can make model construction easier and does not
+require you to define the `forward` method; however, directly inheriting from
+its parent class, `Block`, can greatly expand the flexibility of model
+construction. For example, implementing the `forward` method means you can
+introduce control flow in the network. 
+
+### Constant parameters
+Now we'd like to introduce the notation of a *constant* parameter. These are
+parameters that are not used when invoking backpropagation. This sounds very
+abstract but here's what's really going on.
+Assume that we have some function
 
 $$f(\mathbf{x},\mathbf{w}) = 3 \cdot \mathbf{w}^\top \mathbf{x}.$$
 
-In
-this case 3 is a constant parameter. We could change 3 to something else, say
+In this case $3$ is a constant parameter. We could change $3$ to something else,
+say
 $c$ via
 
 $$f(\mathbf{x},\mathbf{w}) = c \cdot \mathbf{w}^\top \mathbf{x}.$$
+
 Nothing has really changed, except that we can adjust the value of $c$. It is
 still a constant as far as $\mathbf{w}$ and $\mathbf{x}$ are concerned. However,
 since Gluon doesn't know about this beforehand, it's worth while to give it a
```

Comment:
Code block below. Comments line width too large, and don't end with period.

Reply:
Rand_weight -> `rand_weight`

Reply:
nd.dot<?

Reply:
these ways of build a network ->  these ways of building a network

Reply:
resembles more a chimera?

Reply:
I was going to revisit the existing markdown but I'll do so in this PR.

[PEP8(https://www.python.org/dev/peps/pep-0008/#comments)] doesn't say don't end in a period - only short (often inline) comments don't end in a period. 

PEP8 technically also says two spaces after a period, but it's inconsistently followed. I would suggest one, but can refactor to 2.

Reply:
Comments line length: done
rand_weight: done
nd.dot: done
these ways: done
chimera: done

Reply:
I will do another pass of this page once this PR is in. It needs some more work.